Foundation/NSArray.swift

```diff
@@ -853,8 +853,8 @@ open class NSMutableArray : NSArray {
         self.setArray(self.sortedArray(compare, context: context))
     }
 
-    open func sort(_ cmptr: Comparator) {
-        self.sort(options: [], usingComparator: cmptr)
+    open func sort(comparator: Comparator) {
```

Comment:
wow is this really how it came out? it really ought to be `sort(using:)` imho by the naming rules...

Reply:
https://developer.apple.com/documentation/foundation/nsmutablearray list all sorts of variants:

`func sort(using: [NSSortDescriptor])`
`func sort(comparator: (Any, Any) -> ComparisonResult)`
`func sort(options: NSSortOptions = [], usingComparator: (Any, Any) -> ComparisonResult)`
`func sort((Any, Any, UnsafeMutableRawPointer?) -> Int, context: UnsafeMutableRawPointer?)`
`func sort(using: Selector)`

Looks like no one could make up their mind :)

Reply:
This is partially from a breakdown in the importer most likely

Reply:
Which importer is that? 

The changes I made to the tests were to make them compile under Darwin Foundation in Xcode, so then I just changed the sclf code to match.